consensus/applytransaction.go

```diff
@@ -37,13 +36,12 @@ func (s *State) applySiacoinOutputs(bn *blockNode, t Transaction) {
 			}
 		}
 
-		scod := SiacoinOutputDiff{
-			New:           true,
+		bn.siacoinOutputDiffs = append(bn.siacoinOutputDiffs, SiacoinOutputDiff{
+			Direction:     DiffApply,
```

Comment:
woo. Only downside was that it made some of the boolean logic more verbose (e.g. the `commitDiff` functions which now use `if scod.Direction == dir`; not terribly clear, but it wasn't in the first place).